src/server/req_delete.c

```diff
@@ -623,26 +742,21 @@ req_deletejob(struct batch_request *preq)
 			if (--preq->rq_refct == 0) {
 				if ((parent = find_job(jid)) != NULL) {
 					if (req_deletejob2(preq, parent) == 2) {
-						preq->rq_ind.rq_deletejoblist.jobid_to_resume = j;
```

Comment:
just being sure, it's safe to remove this right? we don't need to keep a track of the jobid to resume from when deferring?

Reply:
Its getting saved earlier